packages/client/package.json

```diff
@@ -2,16 +2,28 @@
   "name": "@ethereumjs/client",
   "version": "0.0.6",
   "description": "EthereumJS client implementation",
+  "license": "MPL-2.0",
+  "author": "Vinay Pulim (v@pulim.com)",
+  "keywords": [
+    "ethereum",
+    "ethereumjs",
+    "client",
+    "blockchain",
+    "light sync",
+    "full sync"
+  ],
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist"
```

Comment:
Is this intentional to exclude `bin` here?

Reply:
yes that folder is not needed anymore since it now only contains the typescript source file. the actual built file we are interested in (that the `package.json` key for `bin: ethereumjs` points to) is now at `dist/bin/cli.js`